packages/react-lowcode/src/codegen/generation/generators/detail/mui-detail-generator.ts

```diff
@@ -742,6 +717,16 @@ export default class MuiDetailGenerator implements DetailGenerator {
     }
   }
 
+  private getLabelValueAttribute(
```

Comment:
let's rename to `createLabelAttribute()`

Reply:
renamed